src/Model/Ticket.java

```diff
@@ -1,81 +1,55 @@
 package Model;
 
+import Model.Enums.ConcertHall;
+import Model.Enums.StadiumSector;
+import View.Printer;
+import org.graalvm.nativeimage.c.function.CMacroInfo;
+
 import java.math.BigDecimal;
 import java.sql.Timestamp;
 import java.text.DecimalFormat;
-import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.time.LocalDateTime;
-import java.time.chrono.ChronoLocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeFormatterBuilder;
-import java.time.format.FormatStyle;
 
-public class Ticket {
-    private static int ticketsCounter = 1;
-    private final LocalDateTime ticketCreationTime = LocalDateTime.now();
+public class Ticket implements ID, Printer {
```

Comment:
follow the correct order: constants, variables, constructors, setter/getters, equals+hashCode, toString

Reply:
Fixed. I refactored the class in proper order: constants, variables, constructor, etc.